apps/studio/src/components/sidebar/core/DatabaseDropdown.vue

```diff
@@ -110,14 +108,16 @@
     },
     methods: {
       ...mapActions({refreshDatabases: 'updateDatabaseList'}),
-      ...mapState({ connectionType: 'connectionType' }),
       async databaseCreated(db) {
         this.$modal.hide('config-add-database')
-        // FIXME: move this comparison to the DialectData
-        if (['sqlite', 'firebird'].includes(this.connectionType)) {
-          const fileLocation = this.selectedDatabase.split('/')
-          fileLocation.pop()
-          const url = this.connectionType === 'sqlite' ? `${fileLocation.join('/')}/${db}.db` : `${fileLocation.join('/')}/${db}`
+        if (this.dialectData.disabledFeatures?.multipleDatabase) {
+          const c = this.usedConfig
+          // We'll need the auth credentials for databases like firebird
+          const username = c.username ?? ''
+          const password = c.password ? `:${c.password}` : ''
+          const host = c.host ? `@${c.host}` : ''
+          const port = c.port ? `:${c.port}` : ''
+          const url = this.connectionType === 'sqlite' ? db : `${this.connectionType}://${username}${password}${host}${port}/${db}`;
```

Comment:
Woooah, we absolutely shouldn't be doing this in the UI -- switching off connectionType is bad, but this connection url creation is worse -- what if we have ssh enabled for the connection? What if a DB driver doesn't follow the standard conventions? This doesn't work for oracle for example.

I'm ok with the SQLite fix, but nee you to make sure we fix this new window function too.

Why doesn't it send the connection ID plus the database name? Something as simple as this will work: `bks://connect?connectionId=1&database=foo`

Reply:
`connectionId` will be `null` if we connect to a db without saving the connection. But I agree that this is the worst way to do this lol.